### PR TITLE
Update readme.md

### PR DESCRIPTION
**Pull request by ThatOneCalculator**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -6,4 +6,14 @@ All natural pine, faux fur and a bit of soho vibes for the classy minimalist.
 
 [![](https://img.shields.io/badge/Rosé%20Pine%20Theme-191724)](https://github.com/rose-pine/rose-pine-theme)
 
+### Instructions
```
Comment (mvllow) [inline readme.md]:
```suggestion
## Usage
```

File: readme.md [hunk 2]
```diff
@@ -6,4 +6,14 @@ All natural pine, faux fur and a bit of soho vibes for the classy minimalist.
...
+
+Just run
+```sh
+hyper i hyper-rose-pine
+```
+
+### Screenshots
```
Comment (mvllow) [inline readme.md]:
```suggestion
## Screenshots
```

File: readme.md [hunk 3]
```diff
@@ -6,4 +6,14 @@ All natural pine, faux fur and a bit of soho vibes for the classy minimalist.
 
 [![](https://img.shields.io/badge/Rosé%20Pine%20Theme-191724)](https://github.com/rose-pine/rose-pine-theme)
 
+### Instructions
+
+Just run
```
Comment (mvllow) [inline readme.md]:
```suggestion
```

File: readme.md [hunk 4]
```diff
@@ -6,4 +6,14 @@ All natural pine, faux fur and a bit of soho vibes for the classy minimalist.
...
+```sh
+hyper i hyper-rose-pine
+```
+
+### Screenshots 
+
+
```
Comment (mvllow) [inline readme.md]:
```suggestion
```

**Review by mvllow — CHANGES_REQUESTED** [review]
may need to rebase, there are merge conflicts but hard to see on mobile

**Comment by ThatOneCalculator** [comment]
Hm. I committed the changes that you requested, not so sure about the conflicts.

**Comment by ThatOneCalculator** [comment]
Ok i looked at it on desktop and its kinda dumb, the conflict is that my readme is new :facepalm:

**Review by mvllow — APPROVED** [review]
thanks!

**Comment by ThatOneCalculator** [comment]
It's approved but not merged. GitHub's conflict system is honestly really dumb sometimes...

Do you want me to close this PR and make a new one?

**Comment by ThatOneCalculator** [comment]
image

**Pull request merged**